content/news/posts/2024/prestashop-news-where-we-are-and-where-we-are-going.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title:  Prestashop News - Where We Are And Where We’re Going
+subtitle: "I don't know what to write here"
```

Comment:
@kpodemski @coffrep Please feel free to help me 😄 I don't know what to put here

Reply:
```suggestion
subtitle: "PrestaShop Project Progress: Reflecting on 2023 and Beyond"
```

maybe something like this